axlearn/common/utils.py

```diff
@@ -102,6 +105,26 @@ def sharding(self) -> jax.sharding.Sharding:
 NestedTensorSpec = Optional[Union[TensorSpec, dict[str, Any]]]
 
 
+def offload_dots_saveble(offload_src, offload_dst):
+    """Extract and combine the policy from save_and_offload_only_these_names and dots_saveable.
```

Comment:
Is this accurate? Seems that `dots_saveable` includes `lax_convolution.conv_general_dilated_p`. Also clarify how it combines `save_and_offload_only_these_names `?